website/pages/en/glossary.mdx

```diff
@@ -4,15 +4,21 @@ title: Glossary
 
 - **The Graph**: A decentralized protocol for indexing and querying data.
 
-- **Query**: A request for data. In the case of The Graph, a query is a request for data from a subgraph that will be answered by an Indexer.
+- **Query**: A request for data. In the case of The Graph, a query is a request for data from a subgraph that an Indexer will answer.
 
 - **GraphQL**: A query language for APIs and a runtime for fulfilling those queries with your existing data. The Graph uses GraphQL to query subgraphs.
 
-- **Endpoint**: A URL that can be used to query a subgraph. The testing endpoint for Subgraph Studio is `https://api.studio.thegraph.com/query/<ID>/<SUBGRAPH_NAME>/<VERSION>` and the Graph Explorer endpoint is `https://gateway.thegraph.com/api/<API_KEY>/subgraphs/id/<SUBGRAPH_ID>`. The Graph Explorer endpoint is used to query subgraphs on The Graph's decentralized network.
+- **Endpoint**: A URL that can query a subgraph. The testing endpoint for Subgraph Studio is `https://api.studio.thegraph.com/query/<ID>/<SUBGRAPH_NAME>/<VERSION>` and the Graph Explorer endpoint is `https://gateway.thegraph.com/api/<API_KEY>/subgraphs/id/<SUBGRAPH_ID>`. The Graph Explorer endpoint is used to query subgraphs on The Graph's decentralized network.
 
-- **Subgraph**: A custom API built on blockchain data that can be queried using [GraphQL](https://graphql.org/). Developers can build, deploy and publish subgraphs to The Graph's decentralized network. Then, Indexers can begin indexing subgraphs to make them available to be queried by subgraph consumers.
+- **Firehose**: A versatile and scalable blockchain data processing solution that makes it easy to process and analyze data from any blockchain, regardless of its protocol or implementation. Specific Firehose instrumentation is required for each blockchain protocol.
 
-- **Hosted service**: A temporary scaffold service for building and querying subgraphs as The Graph's decentralized network is maturing its cost of service, quality of service, and developer experience.
+- **Substreams**: Allows developers to define the data Firehose should retrieve from the blockchain and apply transformations to it.
```

Comment:
I feel like this definition is missing a noun, i.e. what is it exactly, not just what it allows?